pkg/pillar/types/downloadertypes.go

```diff
@@ -219,16 +218,6 @@ type DatastoreContext struct {
 
 // AllowNonFreePort looks at GlobalConfig to determine which policy
 // to apply for the download of the object.
-func AllowNonFreePort(gc ConfigItemValueMap, objType string) bool {
-
-	switch objType {
-	case AppImgObj:
-		return gc.GlobalValueTriState(AllowNonFreeAppImages) == TS_ENABLED
-	case BaseOsObj:
-		return gc.GlobalValueTriState(AllowNonFreeBaseImages) == TS_ENABLED
-	default:
-		logrus.Fatalf("AllowNonFreePort: Unknown ObjType %s\n",
-			objType)
-		return false
-	}
+func AllowNonFreePort(gc ConfigItemValueMap) bool {
+	return gc.GlobalValueTriState(AllowNonFreeAppImages) == TS_ENABLED
```

Comment:
*If* we are going to to this then we should rename the field to not include "app" in the name, and remove the Base variable.

Reply:
I agree. I was concerned about everything else upstream that might be affected by it, including both open-source and (especially) commercial controllers. I also was concerned with synchronizing versions and compatibility issues.

I suggest we get this in and working as is, using just the `app` one for both, and then a separate PR that focuses just on eliminating the misnaming of the var.

Reply:
@deitch No controller knows about the AllowNonFreeAppImages name; that is local to EVE.
In theory a test might use the configitems
nordmark@bobo:~/master/eden$ grep -r network.allow.wwan.app.download
nordmark@bobo:~/master/eden$ grep -r network.allow.wwan.baseos.download
but the greps come up empty including on the commercial controller side.
So please rename and remove the baseosos including from CONFIG-PROPERTIES.md

Reply:
@deitch any update on this?

Reply:
Updated.